src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ContactDetails from '../Footer/ContactDetails';
+import AccordionItem from './AccordionItem';
+import './Accordion.scss';
+
+const CN = 'accordion';
+
+class Accordion extends Component {
+  state = {
+    hide: true,
+    currentIndex: null,
+  };
+
+  clickHandler(index) {
+    const { currentIndex } = this.state;
+
+    index === currentIndex ?
+      this.setState(({ hide }) => {
+        return { hide: !hide };
+      }) :
+      this.setState({
+        hide: false,
+        currentIndex: index,
+      });
+  }
+
+  render() {
+    const { hide, currentIndex } = this.state;
+    const { data } = this.props;
+
+    const accordionItems = data.map(({ id, title, description }, index) => (
+      <AccordionItem
+        key={id}
+        title={title}
+        description={description}
+        hide={index !== currentIndex || (index === currentIndex && hide)}
+        onClick={this.clickHandler.bind(this, index)}
```

Comment:
1. binding should be at the top of the component
2. this.clickHandler = this.clickHandler.bind(this)